templates/matchingDropDown.jsx

```diff
@@ -171,7 +171,8 @@ export default function MatchingDropDown(props) {
         disabled={!_isEnabled}
         className={classes([
           'dropdown__btn js-dropdown-btn',
-          !_isEnabled && 'is-disabled'
+          !_isEnabled && 'is-disabled',
+          (displayActiveOption.text !== placeholder) && 'is-selected'
```

Comment:
Text comparison is a poor substitute for the available, more absolute boolean:

```js
const hasActiveOption = Boolean(options.find(option => (option._itemIndex === _itemIndex) && option._isActive));
```
I think that's what you want? `displayActiveOption` is confused by `_isCorrectAnswerShown` and `{ text: placeholder }`.

```js
hasActiveOption && 'is-selected'
```

Reply:
@oliverfoster Thanks, Ollie. I've made this change in the template, and it still seems to work.